src/docsfy/api/projects.py

```
@@ -645,6 +656,29 @@ async def _run_generation(
+        # Persist generation cost and re-sync so clients see the updated value
+        if cost_acc.total_cost_usd > 0:
+            try:
+                await add_generation_cost(
+                    project_name,
+                    ai_provider,
+                    ai_model,
+                    branch=branch,
+                    owner=owner,
+                    cost_usd=cost_acc.total_cost_usd,
+                )
+            except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)